regtests/bin/matrix_cmake_ncep

```diff
@@ -60,51 +60,51 @@ EOF
   modbacio='bacio/2.4.1'
   modg2='g2/3.4.5'
   modw3emc='w3emc/2.10.0'
-  modesmf='esmf/8.5.0'
+  modesmf='esmf/8.6.0'
   modscotch='scotch/7.0.4'
 
 # Set batchq queue, choose modules and other custom variables to fit system and
 # to define headers etc (default to original version if empty)
   ishera=`hostname | grep hfe`
-  isorion=`hostname | grep Orion`
+  isorion=`hostname | grep orion`
   ishercules=`hostname | grep hercules`
   if [ $ishera ]
   then
     batchq='slurm'
     if [ $compiler = "intel" ]
-    then 
+    then
       spackstackpath='/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/modulefiles/Core'
       modcomp='stack-intel/2021.5.0'
       modmpi='stack-intel-oneapi-mpi/2021.5.1'
       metispath='/scratch1/NCEPDEV/climate/Matthew.Masarik/waves/opt/hera/intel/spack-stack/1.6.0/parmetis-4.0.3/install'
       modcmake='cmake/3.23.1'
     elif [ $compiler = "gnu" ]
-    then 
+    then
       spackstackpath='/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/modulefiles/Core'
       modcomp='stack-gcc/9.2.0'
       modmpi='stack-openmpi/4.1.5'
       metispath='/scratch1/NCEPDEV/climate/Matthew.Masarik/waves/opt/hera/gnu/spack-stack/1.6.0/parmetis-4.0.3/install'
       modcmake='cmake/3.23.1'
-    else 
-      echo "Compiler $compiler not supported on hera" 
-      exit 1 
-    fi 
+    else
+      echo "Compiler $compiler not supported on hera"
+      exit 1
+    fi
   elif [ $isorion ]
   then
     if [ $compiler = "intel" ]
     then
       batchq='slurm'
-      spackstackpath='/work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core'
-      modcomp='stack-intel/2022.0.2'
-      modmpi='stack-intel-oneapi-mpi/2021.5.1'
+      spackstackpath='module use  /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env-rocky9/install/modulefiles/Core'
```

Comment:
@MatthewMasarik-NOAA module use should be removed from this statement here and it just be: 
spackstackpath='/work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env-rocky9/install/modulefiles/Core'

Reply:
Oh my.  Thanks for catching that typo @JessicaMeixner-NOAA.

Reply:
I did test and it works, but it'll be good to clean this up all the same. 

Also I was able to replicate your results with the odd tp2.1 diffs.

Reply:
Yep, agreed.

Reply:
Fixed.  Tests re-submitted.